### PR TITLE
Adding simulation variables to the extension gem

### PR DESCRIPTION
**Pull request by TMaile**
This resolves the following task:

["Include simulation variables within the OpenStudio Extension Gem"](https://github.com/BuildingSync/BuildingSync-gem/issues/13)

### CODE REVIEW

**Comment by macumber** [comment]
@nllong can you give us a style review?  Should these be all caps?

**Comment by macumber** [comment]
@TMaile @mrigeshgupta is there an example of these being used anywhere?  any tests?

File: lib/openstudio/extension.rb [hunk 1]
```diff
@@ -41,6 +41,24 @@ module Extension
     class Extension
       attr_accessor :root_dir
 
+      # max number of datapoints to run
```
Comment (nllong) [inline lib/openstudio/extension.rb]: If these are constant class variables, then yeah, i guess they would be all caps.

File: lib/openstudio/extension.rb [hunk 2]
```diff
@@ -41,6 +41,24 @@ module Extension
...
+      MAX_DATAPOINTS = Float::INFINITY
+
+      # number of parallel jobs
+      NUM_PARALLEL = 7
+
+      # do simulations
+      DO_SIMULATIONS = false
+
+      # specify to run the baseline simulation only or not
+      SIMULATE_BASELINE_ONLY = false
```
Comment (macumber) [inline lib/openstudio/extension.rb]: @TMaile SIMULATE_BASELINE_ONLY, DO_MODEL_CALIBRATION , and DO_GET_RESULTS are specific to BuildingSync gem and should be defined there.  They aren't used in Extension Gem

File: lib/openstudio/extension.rb [hunk 3]
```diff
@@ -41,6 +41,24 @@ module Extension
     class Extension
       attr_accessor :root_dir
 
+      # max number of datapoints to run
+      MAX_DATAPOINTS = Float::INFINITY
```
Comment (macumber) [inline lib/openstudio/extension.rb]: @TMaile MAX_DATAPOINTS, NUM_PARALLEL, and DO_SIMULATIONS should be used in [run_osws](https://github.com/NREL/openstudio-extension-gem/blob/develop/lib/openstudio/extension/runner.rb#L625) right?

**Comment by macumber** [comment]
We also want a variable to control if `--verbose` is passed to the OpenStudio CLI in the extension gem

**Pull request merged**